docs/demo/gaiden-css/generic/headings/demo.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="maximum-scale=1,width=device-width,initial-scale=1,user-scalable=0">
+ <link rel="stylesheet" href="/gaiden-css/gaiden.css">
+  <link rel="stylesheet" href="/gaiden-css/base.css">
+  <style>
+    nav {
+      padding: 20px;
+    }
+
+  </style>
+  <title>Headings</title>
+</head>
+
+<body>
+    <h1 class="" arial-level="1" role="heading">
+      H1 Title Gaiden
+      <small>Sub Title Gaiden</small>
```

Comment:
@MarcoBrunoBR also changing a tag is a big refactor. You would have to change every occurency of all titles inside every project that uses Gaiden.